FF119 Add PublicKeyCredential.parseCreationOptionsFromJSON() (#29436)
mdn/content: files/en-us/web/api/credentialscontainer/create/index.md

    - `name`: A string providing a human-friendly identifier for the user's account, to help distinguish between different accounts with similar `displayName`s. This could be an email address (such as `"john.doe@example.com"`), phone number (for example `"+12345678901"`), or some other kind of user account identifier (for example `"johndoe667"`).

> - `hints` {{optional_inline}}

Author: FYI This was missing from the create/get. It is in the IDL for Firefox so expect is implemented.

>   - : A array of "hint" strings that to guide the user-agent in what sort of UI it might use to interact with the user.

Author: A variation because it isn't a UI that the user authenticates

```suggestion
  - : An array of strings providing hints as to what authentication UI the user-agent should provide for the user.
```

>   - : A array of "hint" strings that to guide the user-agent in what sort of UI it might use to interact with the user.

Reviewer: Yup, makes sense.

>     The value can be one of the following:

Author: ```suggestion
    The values can be any of the following:
```
